### ISSUE TITLE
Paulina Introduction and workflow access

### ISSUE THREAD

**Issue by Alviks** [Alviks/Paulina_Graphic_design#1]
Hello Paulina!
I give you some info now so you have some materials to work with. 
But before I do that, i'll orient to you to the work spaces i use for the workflow. 

This page here is called Github - its where I can submit new ticket items to take on this project. You yourself can issue new ticket items if you want me to address or solve other items that are not related. 

You will need to create a login for github so you can follow these ticket items.
[https://github.com/Alviks/Paulina_Graphic_design/issues](url)

I will also give you a folder on the server so you can upload or store, and upload to. 
https://drive.google.com/drive/folders/1I8TzBm89lYvcKxC893Ouv8sQv5DeHOgP?usp=sharing

**Comment by paulinazbylut** [comment]
[projektt.pdf](https://github.com/Alviks/Paulina_Graphic_design/files/2758919/projektt.pdf)
Hello
I've created folder on your google disc and uploaded some projects of logo.
There are 3 first options, each one can be reworked and refined.
I've skipped the moodboard stage because I know it's too late so I did the research on my own (sorry for that, its been a hard time form me theese couple weeks, maybe you've heard what happened)... does not matter right now.

The last project from the pdf contains a little wing drawing (it's not good for now but I think that its quite good association for your profession (  ;) because you in in a certain sense flying) thats why I didn't remove it though I don't really like it for now - I have to work on it If you'd like the idea.

What do you think about the sketches?

Warms
P





[projektt.pdf](https://github.com/Alviks/Paulina_Graphic_design/files/2758920/projektt.pdf)

**Comment by Alviks** [comment]
Most guys like the black and white first one. 

some comments:[https://docs.google.com/presentation/d/1I8vvUfwBhaPJZae8MIaiYcwcC-ZVueAfAy8iYtTTUAs/edit?usp=sharing](https://docs.google.com/presentation/d/1I8vvUfwBhaPJZae8MIaiYcwcC-ZVueAfAy8iYtTTUAs/edit?usp=sharing)

**Comment by paulinazbylut** [comment]
Hello :)
There is a new file on the google disc (project_1opt.PDF
[project_1opt.pdf](https://github.com/Alviks/Paulina_Graphic_design/files/2772898/project_1opt.pdf)
)
Take a look.

WARMS
p

**Comment by Alviks** [comment]
Yes, this is great..that gives us something to bring to the table to Daniel when we meet in Mid Feb. We see what he thinks of the options and if he likes/does not like or would recommend any tweaks he is comfortable with. 

In the meantime, until the meeting. If you get more ideas or make any variations on anything you think can be interesting, send me a comment if you want us to look at something more and we will do so. 

;)

Eddie

**Comment by Alviks** [comment]
Gregor had a good idea. we liked the circle that was around the black and white version. maybe we could look at a colored version w a circle around it. and/or he also suggested how about use the text in an arc around the logo. spreading the text above and below.. such as "Alviks Maleri" above and "Klatter" below to see what is could look like?

**Comment by paulinazbylut** [comment]
Sure, No problem.
I wilk try to do that tomorrow morning.

**Comment by paulinazbylut** [comment]
Hello!
I just added new file to you google disc (the one with today's date)
On the third page there are couple new round options of the logo.
Take a look ;)

Warms / P

**Comment by Alviks** [comment]
Great.. we have a look at it with the guys ;)

**Comment by Alviks** [comment]
Good Morning Paulina, 
So the guys have settled on and voted for the last design. although I will bring a couple more w me when i pitch it to Daniel the big boss. I thought maybe we could also  include one that has a white circle around the logo rather than the black one to have a variation.. and maybe white letters if we are going to be printed on dark clothing. what do you think?
![avk](https://user-images.githubusercontent.com/40559863/51799525-fae3a980-2221-11e9-9856-7579ab684ba0.png)

In the meantime.. can you give me your opinion on placement and size of the logo if we put it on a work t-shirt.. I don't know if we go a standard upper chest area or do we put it on the sleeve or the back of collar.. and how big it should be

Same for the pants.. they are a dark blue as you can see in the image.. we want to figure out where to put the logo and also how big.. have you any opinions on this? 
![pants](https://user-images.githubusercontent.com/40559863/51799564-aee53480-2222-11e9-8395-1b7f28a91791.jpg)
